linchpin/provision/roles/libvirt/tasks/copy_image_src_local.yml

```diff
@@ -0,0 +1,105 @@
+---
+- name: "set local_image_path"
+  set_fact:
+    local_image_path: "{{ libvirt_image_path }}"
+
+- name: "Inside copy image src "
+  debug:
+    msg: "{{ local_image_path  }}  {{ libvirt_image_path }}"
+
+
+- name: "delegate_to localhost :ensure local_image_path directory exists"
```

Comment:
This should have a `become: yes` on it for non-root users.

Reply:
@herlo fixed